software/winrm/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@bostko This was discussed on the mailing list on the Subject: "[PROPOSAL] Split brooklyn-core during OSGification" thread. The ./software/winrm location was suggested by @ahgittin and it's a tad less than ideal, because software/based now depends on software/winrm, but it's good enough.

This change is good regardless because it allows splitting the brooklyn-core into services that are optional (like support for winrm in this case), making it more modular and more manageable, but it's important because the current implementation is blocking the effort to fully OSGi-ify brooklyn, dragging in jython via winrm4j. You are right that a rewrite of winrm4j should be OSGi compliant, but that still leaves the issue of the brooklyn 'glue' which should still be a separate component.